packages/e2e/cypress/plugins/index.js

```diff
@@ -8,7 +8,7 @@
 // ***********************************************************
 
 const cucumber = require('cypress-cucumber-preprocessor').default;
-const resolve = require('resolve');
+const path = require('path');
```

Comment:
This was a known bug whenever `yarn.lock` was updated -- https://github.com/TheBrainFamily/cypress-cucumber-preprocessor/pull/658/files